### ISSUE TITLE
Issue with Label Smoothing Implementation

### ISSUE THREAD

**Issue by ngoc-nguyen-0** [LukasStruppek/Plug-and-Play-Attacks#13]
Hi Lukas,

Thank you for sharing the code.

I'm currently trying to replicate the results from the Label Smoothing paper using the default Facescrub/ResNet152 setup and a smoothing factor of -0.05. However, I'm only achieving a natural accuracy of 80%.

Are there any specific points I should consider when training a model with label smoothing?

Any insights you can provide would be greatly appreciated.

Thank you,
Ngoc

**Comment by LukasStruppek** [comment]
Hi Ngoc,

one important part is the warm-up scheduler for negative smoothing parameters. It is implemented in https://github.com/LukasStruppek/Plug-and-Play-Attacks/blob/bb080ca8c65cacd92847922779f3e418708ff21a/train_model.py#L51 and should be activated automatically when using a negative smoothing factor. However, maybe re-check that there is no implementation error here.

Besides, I uploaded the model weights for the ResNet-152 models trained with negative smoothing here on Github with the latest Release. Maybe you can also use these models for your experiments.

Unfortunately, I am currently traveling and cannot check if the code is working correctly. But I will try it at a later point.

Best,
Lukas

**Comment by ngoc-nguyen-0** [comment]
Hi Lukas,

Thank you for your reply. 
I fixed it. ls_scheduler is defined but it is not used in target_model.fit().

Ngoc

**Comment by LukasStruppek** [comment]
Thanks for pointing out, I will fit it in the code :)
